### DISCUSSION TITLE
Row order changes after a POST request

### DISCUSSION THREAD

**Discussion by Aizen-sama214** [supabase/supabase#9985] [Questions]
Everytime I modify a row using POST request, the modified row goes down, I don't want that to happen. Any suggestions how to achieve that ?

**Comment by GaryAustin1** [comment]
There is no default order in Postgres and it can change between calls.  If you want a specific order you need to order on a column, probably a timestamp.

**Reply by Aizen-sama214** [reply]
So, I need to mention the order request in the api call ? 
If yes, how can I do that ?

**Reply by GaryAustin1** [reply]
https://supabase.com/docs/reference/javascript/order
